docs-new/docs/graphql/core/migrations/manage-metadata.mdx

```diff
@@ -171,70 +133,46 @@ request to replace the metadata. If an admin secret is set, add
 `-H 'X-Hasura-Admin-Secret: <your-admin-secret>'` as the API is an
 admin-only API.
 
-</div>
+</TabItem>
 
-</div>
+</Tabs>
 
-<div class="note">
-
-<div class="title">
-
-Note
-
-</div>
+:::info Note
 
 All the dependent objects, like tables, views, functions etc. should
 exist on Postgres before importing the metadata. Otherwise, it will
 result in an error saying the object does not exist. So, apply the
 Postgres schema first, before importing the metadata.
 
-</div>
+:::
 
 ## Reloading Hasura metadata
 
 In some cases, the metadata can be out of sync with the Postgres schema.
 For example, when a new column has been added to a table via an external
 tool such as `psql`.
 
-<div class="rst-class">
-
-api_tabs
-
-</div>
-
-<div class="tabs">
-
-<div class="tab">
-
-CLI
-
-Metadata can be reloaded with the
-`hasura metadata reload <hasura_metadata_reload>` command.
-
-</div>
-
-<div class="tab">
-
-Console
-
-1.  Click on the settings (⚙) icon at the top right corner of the
-    console screen.
+<Tabs className="api-tabs">
 
-2.  Click on `Reload` button.
+<TabItem value="cli" label="CLI">
+Metadata can be reloaded with the <Link to="/graphql/core/hasura-cli/hasura_metadata_reload.mdx">hasura metadata reload</Link> command.
+</TabItem>
+<TabItem value="console" label="Console">
+1.Click on the settings (⚙) icon at the top right corner of the
```

Comment:
please leave an empty space as first and last line when adding markdown with in the TabItem components.

Also, please add a space between 1.` `and the content for this and for all other usages

Reply:
Cool. but please ensure an empty first line and last line is added for mdx content  inside the TabItem.

Else the content won't be parsed as markdown.